packages/FormElement/stories/examples/Checkbox.js

```diff
@@ -1,34 +1,33 @@
 import React from "react";
-import * as constants from "@paprika/constants/lib/Constants";
+import { action } from "@storybook/addon-actions";
+import L10n from "@paprika/l10n";
 import Checkbox from "@paprika/checkbox";
-import styled from "styled-components";
-import stylers from "@paprika/stylers";
 import { Fieldset } from "../../src";
 
-const { Label, Description, Content } = Fieldset;
+const { Label, Instructions, Content } = Fieldset;
 
 export default function CheckboxExample() {
-  const isDisabled = false;
-  const size = constants.size.MEDIUM;
-  const optionsArray = ["Black Panther", "Spiderman", "Wonder Woman"];
+  const checkboxOptions = ["Black Panther", "Spiderman", "Wonder Woman"];
 
-  const getCheckboxOptions = a11yProps =>
-    optionsArray.map(hero => (
-      <Checkbox key={hero} onChange={() => {}} isDisabled={isDisabled} size={size}>
+  const handleChange = activeIndex => () => {
+    action(`Checkbox ${activeIndex + 1} was clicked.`)();
+  };
+
+  const renderCheckboxes = a11yProps =>
+    checkboxOptions.map((item, index) => (
+      <Checkbox key={item} onChange={handleChange(index)}>
         <Checkbox.Input {...a11yProps} />
-        {hero}
+        {item}
       </Checkbox>
     ));
 
-  const CheckboxGroup = styled.div`
-    margin-top: ${stylers.spacer(2)};
-  `;
-
   return (
-    <Fieldset>
-      <Label>Form Label</Label>
-      <Description>A description</Description>
-      <Content>{a11yProps => <CheckboxGroup>{getCheckboxOptions(a11yProps)}</CheckboxGroup>}</Content>
-    </Fieldset>
+    <L10n>
+      <Fieldset>
+        <Label>Checkbox group label</Label>
+        <Instructions>Some instructions for these checkboxes.</Instructions>
+        <Content>{a11yProps => renderCheckboxes(a11yProps)}</Content>
```

Comment:
Why no CheckboxGroup? I think previously it was rendering too close vertically

Reply:
I think it just adds noise to the example.  The primary purpose is to convey how to use the component, not make it look beautiful.